test/client/actions/admin_scripts.test.js

```diff
@@ -84,21 +102,97 @@ describe('script actions', () => {
       mock.onGet('/scripts').reply(200, scriptListFixtures
       );
     });
+
     afterEach(() => {
       mock.reset();
     });
+
     describe('axios GET request: ', () => {
       it('should add the appropriate action to the store: ', () => {
         const expectedAction = setScriptsList(scriptListFixtures);
+
         return store.dispatch(fetchAllScripts())
           .then(() => {
             const dispatchedActions = store.getActions();
             const { type, payload } = dispatchedActions[0];
+
             expect(dispatchedActions[0]).toEqual(expectedAction);
             expect(type).toBe('SET_SCRIPTS');
             expect(payload).toEqual(scriptListFixtures);
           });
       });
     });
   });
+
+  describe('setScriptQuestions: ', () => {
+    const setScriptQuestionsResult = setScriptQuestions(questionListFixtures);
+
+    describe('type: ', () => {
+      const { type } = setScriptQuestionsResult;
+
+      it('should have a type property (not undefined): ', () => {
+        expect(type).toBeDefined();
+      });
+      it('should have the type property "SET_SCRIPT_QUESTIONS"', () => {
```

Comment:
I followed this pattern based on someone else's test of a similar action. I didn't attempt to rationalize it at the time, but now that I am, I think I agree with you. I imagine some people might view it as a negative test but when you assert that it is equal to a specific thing, I think you can say that you're also asserting that its not equal to other different things.